packages/contracts-bedrock/scripts/verify-foundry-install.ts

```diff
@@ -1,8 +1,23 @@
+import { execSync } from 'child_process'
+
 import { sync as commandExistsSync } from 'command-exists'
 
 if (!commandExistsSync('forge')) {
   console.error(
-    'Command failed. Is Foundry not installed? Consider installing via `curl -L https://foundry.paradigm.xyz | bash` and then running `foundryup` on a new terminal. For more context, check the installation instructions in the book: https://book.getfoundry.sh/getting-started/installation.html.'
+    `Command failed. Is Foundry not installed?
+Consider installing via \`curl -L https://foundry.paradigm.xyz | bash\` and then running \`foundryup -C da2392e58bb8a7fefeba46b40c4df1afad8ccd22\` on a new terminal.
+For more context, check the installation instructions in the book: https://book.getfoundry.sh/getting-started/installation.html.`
   )
   process.exit(1)
 }
+
+const version = execSync('forge --version').toString()
+
+
+if (!version.includes('da239')) {
+  console.warn(
+    `Detected forge version ${version}. This may work, but if you run into any issues, please upgrade to forge 0.2.0 or higher.
```

Comment:
```suggestion
    `Detected forge version ${version}. This may work, but if you run into any issues, please upgrade to forge da2392e58bb8a7fefeba46b40c4df1afad8ccd22.
```

Reply:
I wonder if we should put this inside of a configuration file somewhere. At the moment, the commit hash of the foundry version we pin is in `ci-builder`, the contracts package README, and now here as well. @maurelian - any issues with us putting it in one spot?

Reply:
If the foundry guys decide to allow for pinning the version within the TOML, we can read it from there, but we may need a temp workaround.

Reply:
The markdown file makes it tough to get it down to below 2 spots.  Best we could do for a markdown file is link out to another file since github doesn't support any fancy import directives.   The ci-builder and this could share though

Reply:
How about: 
1. create a file `echo da2392e58bb8a7fefeba46b40c4df1afad8ccd22 > .forge-version`
2. I think that this script and the ci-builder could just read that file, and the markdown could like to it.

Reply:
I like this.   The alternative to linking to it would be to write a very simple ci check that checks that readme has the forge version in it